docs/performance-tuning.md

```
@@ -0,0 +1,270 @@
+## Part 1 — L2 tuning (inter-kernel schedule)
+
+### Capture
+
+Run the case with `--enable-l2-swimlane`. The runtime writes per-task L2
+records and a merged swimlane JSON under the build directory:
+
+```bash
+python models/qwen3/14b/qwen3_14b_decode.py -p a2a3 -d 0 --enable-l2-swimlane
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)